web/src/pages/billing/BillingCurrentCost.tsx

```diff
@@ -190,7 +214,15 @@ const BillingCurrentCost = () => {
                                         case 'field':
                                             return (
                                                 <SUITable.Cell>
-                                                    <b>{p[k.category]}</b>
+                                                    <b>
+                                                        <Link
```

Comment:
Can you pull out this link as a specific component (eg: `BillingCostByTimeLink`) to unify any links and a single place to update if the URL changes.